.github/workflows/publish.yml

```diff
@@ -5,18 +5,14 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    strategy:
```

Comment:
We don't want to run the publish step multiple times, so just one one Node.js version.